subsys/net/lib/lwm2m/lwm2m_obj_server.c

```diff
@@ -240,6 +293,32 @@ static struct lwm2m_engine_obj_inst *server_create(uint16_t obj_inst_id)
 			  transport_binding[index], TRANSPORT_BINDING_LEN);
 	INIT_OBJ_RES_EXECUTE(SERVER_REG_UPDATE_TRIGGER_ID, res[index], i,
 			     update_trigger_cb);
+#if defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1)
+	INIT_OBJ_RES_EXECUTE(SERVER_BOOTSTRAP_UPDATE_TRIGGER_ID, res[index], i,
+			     bootstrap_trigger_cb);
+	INIT_OBJ_RES_OPTDATA(SERVER_APN_LINK_ID, res[index], i, res_inst[index], j);
+	INIT_OBJ_RES_OPTDATA(SERVER_TLS_DTLS_ALERT_CODE_ID, res[index], i, res_inst[index], j);
+	INIT_OBJ_RES_OPTDATA(SERVER_LAST_BOOTSTRAPPED_ID, res[index], i, res_inst[index], j);
+	INIT_OBJ_RES_OPTDATA(SERVER_REGISTRATION_PRIORITY_ORDER_ID, res[index], i, res_inst[index],
+			     j);
+	INIT_OBJ_RES_OPTDATA(SERVER_INITIAL_REGISTRATION_DELAY_TIMER_ID, res[index], i,
+			     res_inst[index], j);
+	INIT_OBJ_RES_OPTDATA(SERVER_REGISRATION_FAILURE_BLOCK_ID, res[index], i, res_inst[index],
```

Comment:
And here `SERVER_REGISTRATION_FAILURE_BLOCK_ID`

Reply:
Ok. I gonna fix that also.